Makefile

```diff
@@ -438,7 +438,7 @@ up-detach: build ## Bring up local Grapl and detach to return control to tty
 	unset COMPOSE_FILE
 	docker-compose \
 		--file docker-compose.yml \
-		up --detach --force-recreate --always-recreate-deps --renew-anon-volumes
+		up --detach --force-recreate --always-recreate-deps --renew-anon-volumes pulumi
```

Comment:
Only run the one container we care about. This also cuts down on the time it takes the tests to run